services/resource-usage-tracker/requirements/_test.in

```diff
@@ -16,6 +16,7 @@ coverage
 docker
 faker
 httpx
+fakeredis[lua]
```

Comment:
interesting! We should perhaps use it everywhere

Reply:
I took inspiration from autoscaling service :+1: there it is already used

Reply:
I took inspiration from the autoscaling service :+1: there it is already used